policy_assume_read_only_role.tf

```
@@ -0,0 +1,22 @@
+# IAM policy document that allows assumption of the WAS DB read-only role in the
+# User Services account
+data "aws_iam_policy_document" "assume_userservices_was_db_read_only_role_doc" {
+  statement {
+    effect = "Allow"
+
+    actions = ["sts:AssumeRole"]
```

If we wanted to alphabetize. Also the `TagSession` permission.

```suggestion
    actions = [
      "sts:AssumeRole",
      "sts:TagSession",
    ]

    effect = "Allow"
```